### ISSUE TITLE
Add Presets

### ISSUE THREAD

**Issue by Banalian** [Banalian/ExtraLeaderboardPositions#6]
I'm not sure yet if it's good or if i should implement it, but having presets could be nice, with a way to switch bewteen them easily, to have maybe a COTD preset or some campaing one or a preset for less populated leaderboards and all

**Comment by tboby** [comment]
If you think the % rankings feature is good, and think having extra positions by top % (not top n) is good, I think this might be quite important. My concern with adding everything is that it'll start to get overwhelming and cluttered (which puts people off). 

People don't tend to look at settings too hard sadly :)

Having an easy preset for "top n" vs "top %" vs "medals only" etc might be nice.

**Comment by Banalian** [comment]
I agree with your point yes, i'll have to look into that again now that the code as evolved so much
